.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -e
+CLI_INSTALL_VERSION='v0.5.5' # reference a particular released version
+echo "#######################################################"
+echo "### Install Velocitas CLI ${CLI_INSTALL_VERSION}    ###"
+echo "#######################################################"
+
+ARH=$(uname -m)
+ARH=${ARH/aarch64/arm64}
+ARH=${ARH/x86_64/x64}
+CLI_ASSET_NAME=velocitas-linux-${ARH}
```

Comment:
```suggestion
CLI_ASSET_NAME=velocitas-linux-$(echo $1 | sed 's/amd64/x64/g')
```

Why did you change this? any specific reason?